CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 35.7.3 [#1081](https://github.com/openfisca/openfisca-core/pull/1081)
+
+- Fix misnamed attribute in mis-sized population error message
```

Comment:
We try to keep the changelog user-oriented and not get too technical in it 🙂 
```suggestion
- Correct error message in case of mis-sized population 
```